openquake/sub/get_profiles_from_slab2pt0.py

```diff
@@ -377,6 +377,7 @@ def get_profiles(fname_str: str, fname_dep: str, spacing: float, fname_fig:
             # fig.basemap(region=reg, projection="M20c", frame=True)
             fig.basemap(region=reg, projection=f"T{clo}/{cla}/12c", frame=True)
             fig.coast(land="gray", water="skyblue")
+            import pdb; pdb.set_trace()
```

Comment:
```suggestion

```
Remove breakpoint

Reply:
Ciao Kendra, one super minor suggestion to remove a breakpoint but otherwise LGTM